src/test/user.spec.js

```diff
@@ -37,7 +42,35 @@ describe('Test user login, signup and account verification', () => {
         });
     });
 
-    it('it should throw an error when user with an unverified account wants to signup', done => {
+    it('should mock a sendgrid api request', async () => {
+      nock('https://api.sendgrid.com')
+        .post('/api/mail.send.json')
+        .reply(200, {
+          message: 'success'
+        });
+
+      const res = await chai
+        .request('https://api.sendgrid.com')
+        .post('/api/mail.send.json');
+      expect(res.body.message).to.include('success');
+      expect(res.status).to.eql(200);
+    });
+
+    it('should mock a failed sendgrid api request', async () => {
+      nock('https://api.sendgrid.com')
+        .post('/api/mail.send.json')
+        .reply(400, {
+          message: 'failed'
+        });
+
+      const res = await chai
+        .request('https://api.sendgrid.com')
+        .post('/api/mail.send.json');
+      expect(res.body.message).to.include('failed');
```

Comment:
I'd prefer that you assert the actual error message not just that it includes a  word or phrase.